src/i18n/locales/ca/common.json

```diff
@@ -96,5 +96,12 @@
 			"groqApiKey": "Clau API de Groq",
 			"getGroqApiKey": "Obté la clau API de Groq"
 		}
+	},
```

Comment:
On my phone but I’m not sure common.json is the best place for these strings. Are the other chat buttons etc in chat.json?

Reply:
I moved these strings out of `core/common` and into `webview-ui/chat`